src/main/java/com/minecolonies/coremod/commands/colonycommands/ColonyTeleportCommand.java

```diff
@@ -1,7 +1,9 @@
 package com.minecolonies.coremod.commands.colonycommands;
 
 import com.minecolonies.api.colony.permissions.Action;
+import com.minecolonies.api.colony.permissions.Rank;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.colony.permissions.Rank'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)